src/pages/home/sidebar/SidebarScreen/sidebarPropTypes.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import {windowDimensionsPropTypes} from '../../../../components/withWindowDimensions';
+import {withLocalizePropTypes} from '../../../../components/withLocalize';
+
+const propTypes = {
+    /* Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
I know this was missing earlier as well, but if you see we're adding `allPolicies` as well in the props via Onyx. So we would need to add it in the `propTypes` too.